test-harness/scenarios/require-module.scenario

```diff
@@ -0,0 +1,34 @@
+====test====
+Custom function is able to require a module
+====document====
+{}
+====asset:fn.js====
+const fs = require('fs');
+const path = require('path');
+const dir = path.join('{asset:fn.js}', '../node_modules', 'foo');
+
+fs.mkdirSync(dir, { recursive: true });
+fs.writeFileSync(path.join(dir, 'package.json'), JSON.stringify({ main: './index.js' }));
+fs.writeFileSync(path.join(dir, 'index.js'), 'module.exports = Function()');
+
+try {
+  require('foo'); // this will throw if foo cannot be resolved
+} finally {
+  fs.unlinkSync(path.join(dir, 'package.json'));
+  fs.unlinkSync(path.join(dir, 'index.js'));
+  fs.rmdirSync(dir);
```

Comment:
@P0lip This doesn't seem to work properly. After the run, a `/tmp/node_modules` folder still exists

Reply:
Hmm, good catch. I'll take a look